Assets/MixedRealityToolkit/Providers/BaseGenericInputSource.cs

```diff
@@ -22,7 +22,7 @@ public BaseGenericInputSource(string name, IMixedRealityPointer[] pointers = nul
         {
             SourceId = (InputSystem != null) ? InputSystem.GenerateNewSourceId() : 0;
             SourceName = name;
-            Pointers = pointers ?? new[] { InputSystem?.GazeProvider.GazePointer };
+            Pointers = pointers ?? new[] { InputSystem?.GazeProvider?.GazePointer };
```

Comment:
seems unrelated, did this change sneak in? or are you fixing a bug you found? Sometimes I do that, too. I normally make a note of small bugs like this that I fix in the PR to help folks in the future.

Reply:
This is a bug I noticed in the editor tests.